Velog/style.css

```
@@ -0,0 +1,329 @@
+body {
+    box-sizing: border-box;
+    background-color: rgb(228, 236, 247); 
+}
+
+div {
+    display: block;
+}
+
+body {
+    margin: auto;
+    display: flex;
+    width: 100%;/*레이아웃 유연하게 지정*/
+    flex-direction: column;
+}
```

```suggestion
body {
    box-sizing: border-box;
    background-color: rgb(228, 236, 247);
    margin: auto;
    display: flex;
    width: 100%;/*레이아웃 유연하게 지정*/
    flex-direction: column; 
}

div {
    display: block;
}

```

같은 태그에 대한 스타일은 하나로 합쳐봅시다  😀

---

헉 넵넵!! 다음부터 하나로 합치도록 하겠습니다!! 감사합니다